resources/bundle_readme.md

```diff
@@ -0,0 +1,57 @@
+Welcome to the napari installation contents
+-------------------------------------------
+
+This is the base installation of napari, a fast n-dimensional image viewer written in Python.
+
+## How do I run napari?
+
+In most cases, you would run it through the platform-specific shortcut we created for your
+convenience. In other words, _not_ through this directory! You should be able to see a
+`napari (x.y.z)` menu item, where `x.y.z` is the installed version.
```

Comment:
What does 
`You should be able to see a `napari (x.y.z)` menu item, where `x.y.z` is the installed version.`
refer to?

Reply:
The shortcut installed in `~/Applications`, for example

Reply:
Ok, is that a linux/windows thing? `menu item` to me means like Edit>Copy or File>Open.
Again, I'm coming from the point of view of reading this README in the conda dir, not in napari or the installer—maybe this is where I'm off?

Reply:
menu item means entry in menu start, launcher, and other lists of installed applications. 
This was introduced to allow people to keep the older version of napari. Sample situation will be keeping the same environment as data was sent to publication until all review process is finished.

Reply:
> Again, I'm coming from the point of view of reading this README in the conda dir

This file will only be included in installations obtained with the conda-based installer. It should explain what this directory is and what you can do with it if you are willing to play in advanced mode.

We can change the wording so "menu item" is not used (I hadn't thought of the toolbar menu meaning). The entity I refer to by "menu item" is a little script that executes `napari` after activating its conda environment. We can call it "launcher shortcut"?

Reply:
To me that makes it much clearer that you are referring to the things that are listed below. Definitely prefer shortcut vs. menu item.

Reply:
> Definitely prefer shortcut vs. menu item.

Me too.